amp/alfresco5/src/main/amp/config/alfresco/module/antiidle/module-context.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version='1.0' encoding='UTF-8'?>
+
+<!DOCTYPE beans PUBLIC '-//SPRING//DTD BEAN//EN' 'http://www.springframework.org/dtd/spring-beans.dtd'>
+<beans>
+
+    <bean id="AntiIdleScheduledJobExecuter"
+            class="eu.xenit.alfresco.AntiIdleScheduledJobExecuter">
+        <property name="serviceRegistry">
+            <ref bean="ServiceRegistry" />
+        </property>
+    </bean>
+
+    <bean id="JobDetail"
```

Comment:
Hmm, this looks like a very generic bean ID to me. High risk a bean with this ID already exists or is also used by other extensions?